app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -234,6 +236,7 @@ public void onFinish() {
   private TextView tabSwitcherIcon;
   private TableDrawerAdapter tableDrawerAdapter;
   private RecyclerView tableDrawerRight;
+  private AutoTransition autoTransition;
```

Comment:
Can we initialize this variable here itself instead of initializing it twice in the methods? Will it work?

Reply:
Yes we can. I made the changes.